api/client/webclient/webclient.go

```diff
@@ -30,10 +30,12 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/gravitational/trace"
-
 	"github.com/gravitational/teleport/api/constants"
 	"github.com/gravitational/teleport/api/defaults"
+	"github.com/gravitational/teleport/api/utils"
+	"github.com/gravitational/trace"
+
+	"log"
```

Comment:
Wrong logger.

Reply:
I _was_ using logrus, but @Joerger asked me to remove it to reduce the API client dependencies.

Reply:
@tcsc @Joerger Looks like using the standard logger here causes tsh to print debug statements in the stdout even without `--debug` flag:

```
➜  ~ tsh login --proxy=root.gravitational.io:3080 --auth=local --user=alice
2021/09/20 15:48:32 Attempting GET root.gravitational.io:3080/webapi/ping/local
Enter password for Teleport user alice:
> Profile URL:        https://root.gravitational.io:3080
  Logged in as:       alice
  Cluster:            root
  Roles:              admin
  Logins:             root, ubuntu
  Kubernetes:         enabled
  Kubernetes groups:  system:masters
  Valid until:        2021-09-21 03:48:37 -0700 PDT [valid for 12h0m0s]
  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
```

Let's either revert back to using logrus or configure std logger to only log these in debug mode.

I would just use logrus - api module already depends on it and I don't see us removing this dependency any time soon and maintaining 2 different loggers is cumbersome.

Reply:
OK. Will revert.